### PR TITLE
Move chaining todo item to "completed"

### PR DESCRIPTION
**Pull request by JakobJingleheimer**

### CODE REVIEW

**Comment by JakobJingleheimer** [comment]
Should we also add "Move loaders off thread" and "Convert `resolve` from async to sync" (and perhaps an accompanying "Add async `resolve` to `module` module"?) to the Upcoming list?

**Comment by arcanis** [comment]
Do you think, with the chaining being merged, it's worth discussing ambient loaders (and perhaps "should ambient loaders just be loaders", as I recall @cspotcode suggested it and we didn't recall why it'd be best avoided)? I would like to try to make a prototype soonish, if there's consensus on the general design.

File: README.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ This team is spun off from the [Modules team](https://github.com/nodejs/modules)
...
 
+1. Implement chaining as described in the [design](doc/design/proposal-chaining-middleware.md), where the `default<hookName>` becomes `next` and references the next registered hook in the chain.
+
 ### In Progress
 
 ### Upcoming
 
-1. Implement chaining as described in the [design](doc/design/proposal-chaining-middleware.md), where the `default<hookName>` becomes `next` and references the next registered hook in the chain.
-
 1. Get a `load` return value of `format: 'commonjs'` to work, or at least error informatively. See https://github.com/nodejs/node/issues/34753#issuecomment-735921348.
```
Comment (GeoffreyBooth) [inline README.md:39]: Isn't this already done too?

Comment (JakobJingleheimer) [reply README.md:39]: I think no because ESMLoader can't talk to CJSLoader or something. We could add an error (or maybe a warning?) though.

Comment (GeoffreyBooth) [reply README.md:39]: If you or someone has time, we should confirm this one way or the other. I think I thought it was broken but then the test CoffeeScript loader got fixed, which proves that this wasn't an issue (but it was an author error on my part, which suggests maybe we need a better error message).

**Pull request merged**